goad

```diff
@@ -71,9 +71,15 @@ else
 		;;
 	test-acceptance)
 		[ -x "$GOPATH/bin/repeatr" ] || { echo "run 'goad install' first" 2>&1 ; exit 19; }
+		# demo and basic acceptance scenarios should run just about anywhere
 		./demo.sh -t
 		./lib/integration/runAll.sh
-		./repeat-thyself.sh
+		# repeat-thyself should usually run, but it has to sit out in travis PR-mode because of the synthetic commit which isn't actually cloneable
+		if [ "$TRAVIS" == 'true' -a "$TRAVIS_PULL_REQUEST" != 'false' ]; then
```

Comment:
I have a preference for `[ "$TRAVIS" == 'true' ] && [ "$TRAVIS_PULL_REQUEST" != 'false' ]` but inconsequential.